app/config/config.yml

```diff
@@ -44,12 +44,12 @@ assetic:
     debug:          "%kernel.debug%"
     use_controller: false
     bundles:        [ ]
-    #java: /usr/bin/java
     filters:
         cssrewrite: ~
-        sass:
-            bin:      "/usr/bin/sass"
-            apply_to: "\.scss$"
+        jsqueeze: ~
+        scssphp:
+            # the formatter must be the FQCN (don't use the 'compressed' value)
```

Comment:
If I use the following config:

```yaml
assetic:
    filters:
        scssphp:
            formatter: 'compressed'
```

I get the following error:

![assetic_error](https://cloud.githubusercontent.com/assets/73419/7085744/0f39c4ee-df7a-11e4-90da-5218306177cf.png)